diff --git a/dist/graph.js b/dist/graph.js
--- a/dist/graph.js
+++ b/dist/graph.js
@@ -86,7 +86,7 @@ export class MdxPath {
                 normalizedPath = join(this.relativeDir, this.filename);
                 urlPath.pathname = normalizedPath;
             }
-            path = parse(normalizedPath);
+            path = parse(join(normalizedPath));
         }
         else {
             try {
